app/views/pages/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<% set_page_title(@form.name) %>
+
+<% content_for :back_link, govuk_back_link_to(form_path(@form), "Back to create a form") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @form.name %></span>
+    <h1 class="govuk-heading-l"><%= t("pages.index.title") %></h1>
+
+    <% if flash[:message] %>
+      <p class="govuk-body"><%= flash[:message] %></p>
+    <% end %>
+
+    <%= link_to t("pages.index.add_question"), new_page_path(@form), class: "govuk-button govuk-!-margin-bottom-3 govuk-!-margin-top-3", 'data-module':"govuk-button", draggable:"false", role: "button" %>
```

Comment:
Could this be a `govuk_button_link_to`?
e.g.
```suggestion
    <%= govuk_button_link_to t("pages.index.add_question"), new_page_path(@form), class: "govuk-!-margin-bottom-3 govuk-!-margin-top-3"%>
```

Reply:
👍